Dockerfile.dapper

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG=golang:1.23.4-alpine3.20
+ARG GOLANG=golang:1.23.4-alpine3.21
```

Comment:
Would be good if the golang version could also be updated here

Reply:
golang should be kept in sync with whatever is needed for the specific k8s version, it should not be bumped arbitrarily to latest.